tools/pycafe/test_pycafe_generated_links.py

```diff
@@ -78,7 +51,7 @@ def test_pycafe_link(url: str, wait_for_text: str):
     url_generated = generate_link(config, dev_directory, extra_requirements, use_latest_release=False)
 
     # Test the link
-    success = test_pycafe_link(url=url_generated, wait_for_text="Vizro Features")
+    success = test_pycafe_link(url=url_generated, wait_for_text="Tips Dashboard", wait_for_locator=False)
```

Comment:
I'd change all this `success` stuff to `exit_code` to match the new exit code system. If you do a search for `success` then feel free to change all of them to match my proposal.

Then e.g. L68 could be removed completely because you could just put the `sys.exit` in the `if exit_code`.